addons/website_profile/views/website_profile.xml

```diff
@@ -296,64 +296,69 @@
                         <p>You can score more points by answering quizzes at the end of each course content. Points can also be earned on the forum. Follow this link to the guidelines of the forum.</p>
                     </div>
                     <div class="col-12 col-md-5 col-lg-4">
-                        <t t-call="website_profile.rank_content"/>
+                        <div class="card">
+                            <div class="card-header border-bottom-0">Ranks</div>
+                            <ul class="list-group list-group-flush">
+                                <t t-foreach="ranks" t-as="rank">
+                                    <t t-set="order_inverse" t-value="6 - rank.id"/>
```

Comment:
`6 - rank.id` ?? What ? I'm gonna need an explanation ^^'

Reply:
Done, I just reverse the order in the controller before sending data to the template.